src/getMigrateDashboards.ts

```diff
@@ -53,6 +60,8 @@ export const getMigrateDashboards =
   ): void => {
     const { content, structure } =
       contentServer.children?.ui.children?.dashboards.children ?? {};
+    const originalContent =
+      originalContentServer.children?.ui.children?.dashboard.children?.content;
```

Comment:
This doesn't work for `--from-version="4.3"`, and same for widgets/filters.
Instead, the caller could pass `originalContent` directly, and in practice it would be defined once and for all in `bin.ts`.